project/project.go

```diff
@@ -70,25 +79,34 @@ func (proj *Project) MergeFrom(mergeProj Project) error {
 			return nil
 		}
 
-		fileContent, err := ioutil.ReadFile(filePath)
+		fileReader, err := os.Open(filePath)
```

Comment:
It might be easier to test if this function was no longer anonymous and took the `fileReader` and `srcFile` `io` things as parameters.